tools/version-tracker/pkg/constants/constants.go

```diff
@@ -307,4 +307,21 @@ var (
 	CiliumImageDirectories = []string{"cilium", "operator-generic", "cilium-chart"}
 
 	ProjectsSupportingPrereleaseTags = []string{"kubernetes-sigs/cluster-api-provider-cloudstack"}
+
+	// These projects will be upgraded only on main and won't be triggered on release branches.
+	UpstreamPackagesProjects = []string{
+		"aquasecurity/harbor-scanner-trivy",
+		"aquasecurity/trivy",
+		"aws-containers/hello-eks-anywhere",
+		"aws-observability/aws-otel-collector",
+		"distribution/distribution",
+		"emissary-ingress/emissary",
+		"goharbor/harbor",
+		"kubernetes/autoscaler",
```

Comment:
include `kubernetes/cloud-provider-aws` here too, as it's a dependency of eks-anywhere-packages.

Reply:
I missed it because it was missed in the prow job PR too: https://github.com/aws/eks-anywhere-prow-jobs/pull/402